packages/react-dev-utils/printBuildError.js

```diff
@@ -23,24 +22,22 @@ module.exports = function printBuildError(err) {
     message.indexOf('from UglifyJs') !== -1
   ) {
     try {
-      const matched = /Unexpected token:(.+)\[(.+):(.+),(.+)\]\[.+\]/.exec(
-        stack
-      );
+      const matched = /(.+)\[(.+):(.+),(.+)\]\[.+\]/.exec(stack);
```

Comment:
There are other errors, so I went for the standard line format from this regex. Seems to work.